### ISSUE TITLE
A typo in the documentation.

### ISSUE THREAD

**Issue by duongna21** [pytorch/ignite#631]
In [QUICKSTART](https://pytorch.org/ignite/quickstart.html#id1), there is a redundant argument at the code

`@trainer.on(Events.ITERATION_COMPLETED)
def log_training_loss(engine):
    print("Epoch[{}] Loss: {:.2f}".format(engine.state.epoch, len(train_loader), engine.state.output))`

whereas it should be

`@trainer.on(Events.ITERATION_COMPLETED)
def log_training_loss(engine):
    print("Epoch[{}] Loss: {:.2f}".format(engine.state.epoch, engine.state.output))`

Additionally, the code 

`@trainer.on(Events.EPOCH_COMPLETED)
def log_training_results(trainer):
    evaluator.run(train_loader)
    metrics = evaluator.state.metrics
    print("Training Results - Epoch: {}  Avg accuracy: {:.2f} Avg loss: {:.2f}"
          .format(trainer.state.epoch, metrics['accuracy'], metrics['nll']))`

`@trainer.on(Events.EPOCH_COMPLETED)
def log_validation_results(engine):
    evaluator.run(val_loader)
    metrics = evaluator.state.metrics
    print("Validation Results - Epoch: {}  Avg accuracy: {:.2f} Avg loss: {:.2f}"
          .format(engine.state.epoch, metrics['accuracy'], metrics['nll']))`

should use `trainer` and `engine` in a consistent way.

**Comment by vfdev-5** [comment]
@duongna21 thanks for the feedback ! Yes, we need to improve the documentation.
